stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -145,12 +145,16 @@ __swift_uint32_t _stdlib_cxx11_mt19937(void);
 SWIFT_RUNTIME_STDLIB_INTERNAL
 __swift_uint32_t _stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound);
 
+// Random number for stdlib
+SWIFT_RUNTIME_STDLIB_INTERFACE
+void _stdlib_random(void *buf, __swift_size_t nbytes);
```

Comment:
Should this function be `SWIFT_RUNTIME_STDLIB_INTERNAL` instead?
Should `@_inlineable` be removed from the `Random.next()` method?
<https://github.com/apple/swift/blob/master/stdlib/public/SwiftShims/Visibility.h>

Reply:
Thanks!